### PR TITLE
Adds GSI to partners

### PR DESCRIPTION
**Pull request by yoavwe**
### Description
Adds GSI to partners

### CODE REVIEW

File: _partners/gsi-technology.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: GSI Technology
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/just-gsi-logo.png'
+link: 'https://gsitechnology.com/apu'
```
Comment (stockholmux) [inline _partners/gsi-technology.md:5]: Humm. That URL doesn't seem to work.

**Review by stockholmux — CHANGES_REQUESTED** [review]
Could you double check the URL - doesn't seem to be working for me.

File: _partners/gsi-technology.md (hunk 1 above)
Comment (yoavwe) [reply _partners/gsi-technology.md:5]: Double checked - https://gsitechnology.com/apu works fine - here is a screenshot 
Screen Shot 2021-07-07 at 11 22 28

Comment (yoavwe) [reply _partners/gsi-technology.md:5]: https://gsitechnology.com/apu seems to be working fine  - here is a screenshot:
Screen Shot 2021-07-07 at 11 22 28

Comment (yoavwe) [reply _partners/gsi-technology.md]: www added

**Comment by elfisher** [comment]
The URL seems to be working for me now. @stockholmux are you still seeing an issue?

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**